src/dinit.cc

```diff
@@ -453,6 +471,89 @@ static int process_commandline_arg(char **argv, int argc, int &i, options &opts)
     return 0;
 }
 
+#if SUPPORT_SELINUX
+// Load the system SELinux policy and transition ourselves to it.
```

Comment:
I recommend changing the commenting style to either use
```
/**
 * documentation comment
 */
``` 
or 
```
/// documentation comment line 1
/// documentation comment line 2
```

See: https://www.doxygen.nl/manual/docblocks.html for additional examples.

Even if doxygen (or any other documentation generator) isn't used, it's a convenient way to denote documentation comments and even more convenient for others to identify such comments.

Reply:
We're not using either for the moment so `//`-style comments is the right choice. I might revisit this at some point but for now I don't want a new style of comment introduced, thanks.

Reply:
Noted